File: Bars.GkhCR/Entities/ObjectCR/Qualification/VoiceMember.cs
namespace Bars.GkhCr.Entities
{
    using System;

    using Bars.Gkh.Entities;
    using Bars.GkhCr.Enums;

    /// <summary>
    /// Голос участника квалификационного отбора
    /// </summary>
    public class VoiceMember : BaseGkhEntity
    {
        /// <summary>
        /// Квалификационный отбор
        /// </summary>
        public virtual Qualification Qualification { get; set; }

        /// <summary>
        /// Участник квалификационного отбора
        /// </summary>
        public virtual QualificationMember QualificationMember { get; set; }

        /// <summary>
        /// Тип принятия голоса квалификационного отбора
        /// </summary>
        public virtual TypeAcceptQualification TypeAcceptQualification { get; set; }

        /// <summary>
        /// Дата документа
        /// </summary>
        public virtual DateTime? DocumentDate { get; set; }

        /// <summary>
        /// Причина
        /// </summary>
        public virtual string Reason { get; set; }
    }
}

File: Bars.GkhCR/libs/B4/model/ObjectCR/Qualification/VoiceMember.js
Ext.define('B4.model.objectcr.qualification.VoiceMember', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'VoiceMember'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'Qualification', defaultValue: null },
        { name: 'QualificationMember', defaultValue: null },
        { name: 'MemberName' },
        { name: 'DocumentDate', defaultValue: null },
        { name: 'Reason' },
        { name: 'TypeAcceptQualification', defaultValue: 10 }
    ]
});